src/docs/sdk/performance/span-data-conventions.mdx

```diff
@@ -12,10 +12,14 @@ Below describes the conventions for the Span interface for the `data` field on t
 
 ## HTTP
 
-| Attribute       | Type   | Description                          | Examples           |
-| --------------- | ------ | ------------------------------------ | ------------------ |
-| `http.query`    | string | The Query string present in the URL. | `?foo=bar&bar=baz` |
-| `http.fragment` | string | The Fragments present in the URL.    | `#foo=bar`         |
+| Attribute                          | Type   | Description                           | Examples           |
+| ---------------------------------- | ------ | ------------------------------------- | ------------------ |
+| `http.query`                       | string | The Query string present in the URL.  | `?foo=bar&bar=baz` |
+| `http.fragment`                    | string | The Fragments present in the URL.     | `#foo=bar`         |
+| `http.method`                      | string | The HTTP method used.                 | `GET`              |
+| `http.request_content_length`      | number | The encoded body size of the request. | `123`              |
+| `http.decoded_request_body_length` | number | The decoded body size of the request. | `456`              |
```

Comment:
Ah, I realized there's a mistake here that I merged in my last PR. I think we should be using `response` and not `request`

Reply:
you're right! I can update.